impl/src/templates/remote_crates.bzl.template

```diff
@@ -6,18 +6,10 @@ DO NOT EDIT! Replaced on runs of cargo-raze
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")
 
-def _new_http_archive(name, **kwargs):
-    if not native.existing_rule(name):
-        http_archive(name=name, **kwargs)
-
-def _new_git_repository(name, **kwargs):
-    if not native.existing_rule(name):
-        new_git_repository(name=name, **kwargs)
```

Comment:
@dfreese  In the old version, this was the generated rule, so I was trying to stay consistent. If they're functionally equivalent, I'm happy to also add a migration to `git_repository`, but just a heads up.

Reply:
gotcha, thanks for the clarification.  Briefly glancing at the docs, I'm not sure I can tell the difference between a rule and it's `new_` equivalent.